pg-auto-failover-enterprise.spec

```
@@ -371,6 +371,9 @@ rm -f %{pginstdir}/bin/pg_autoctl
 
 
 %changelog
+* Fri Sep 17 2021 - Gurkan Indibay <gindibay@microsoft.com> 1.6.2-2
+- Official 1.6.2 release of Pg Auto Failover Enterprise
```

We can have a better description here for future reference.

```suggestion
- Removes encryption from packages
```

---

Fixed